src/test/java/com/waverleysoftware/BaseClass.java

```diff
@@ -0,0 +1,25 @@
+package com.waverleysoftware;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.Reporter;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+public class BaseClass {
+    WebDriver driver;
+
+    @BeforeClass
+    public void setupApplication() {
+        Reporter.log("Session started", true);
+        driver = new FirefoxDriver();
+        driver.get("https://www.facebook.com/");
+        Reporter.log("Application started", true);
+    }
+
+    @AfterClass
+    public void closeApplication() {
+        driver.quit();
+        Reporter.log("Browser session end", true);
+    }
+}
```

Comment:
new line is missing at the end of a file... make sure that all java classes have new line at the end...